docs/source/AdministratorGuide/Tutorials/installWMS.rst

```diff
@@ -0,0 +1,340 @@
+                queue
+                {
+                  CPUTime = 40000
+                  MaxTotalJobs = 5
+                  MaxWaitingJobs = 10
+                  BundleProxy = True
+                  BatchError = /home/diracpilot/localsite/error
+                  ExecutableArea = /home/diracpilot/localsite/submission
+                  RemoveOutput = True
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+We set the type of the CE, ``SSH`` in our case, as well as the required parameters to access the Element. 
+Then we configure the queue that is going to receive the jobs. A queue corresponds to a set of Worker Nodes in practice.
+
+Note: make sure the ``CPUTime`` of the queue is above the ``CPUTime`` of the job, else the job will not be scheduled to run on this Worker Node.
+
+Configuring the pilots
+======================
+
+A job is not able to run directly on a Worker Node and needs to be executed by a pilot that has the knowledge of its environment and knows how to run jobs within it.
+The pilot is the first job to be deployed on a Worker Nde and it installs, configures DIRAC and asks for pending jobs in Task Queues that would match the environment of the Worker Node. Add the following lines in the ``/Operation/MyDIRAC-Production`` section using the WebApp::
+
+  Pilot
+  {
+    Version = v7r0p36
+    CheckVersion = False
+    Command
+    {
+      Test = GetPilotVersion
+      Test += CheckWorkerNode
+      Test += InstallDIRAC
+      Test += ConfigureBasics
+      Test += ConfigureCPURequirements
+      Test += ConfigureArchitecture
+      Test += CheckCECapabilities
+      Test += LaunchAgent
+    }
+    GenericPilotGroup = dirac_user
+    GenericPilotUser = ciuser
+    pilotFileServer = dirac-tuto:8443
+  }
+
+We pass our credentials information to the pilot so that it can interact with DIRAC as it needs to execute the commands defined in ``Commands``.
+Only a small script called ``pilotWrapper`` is directly passed to the CE, most of the files used by the pilot will be downloaded from ``pilotFileServer`` during the script execution.
+These files can be uploaded and updated at each commit done to the configuration, we just need to create the directory that is going to contain the files required by the pilot and add the information within the configuration. First, add the option below to the configuration, in the ``/WebApp`` section::
+
+    StaticDirs = pilot
+
+Due to installation issue, the synchronization CS-Pilot is disabled by default, modify the ``/Operations/Defaults/Pilot`` section in the ``/opt/dirac/etc/dirac.cfg`` file as ``dirac``::
+
+    UpdatePilotCStoJSONFile = True
+
+Still as ``dirac``, create the pilot repository that will contain all the pilot files that will be updated whenever a CS update is triggered::
+
+    mkdir -p /opt/dirac/webRoot/www/pilot
+
+
+Note: Do not put the Pilot configuration in ``Operation/Defaults``, DIRAC would not be able to get it.
```

Comment:
Why is that?

And
```suggestion
.. warning:: Do not put the Pilot configuration in ``Operation/Defaults``, DIRAC would not be able to get it.
```

Reply:
I didn't remember so I checked again, and this is very strange.
When I put the `Pilot` configuration into `Operation/Defaults`, then the Site Director is not able to fetch parameters to built the Pilot options.
If I set a breakpoint, it is able to get them. 
Is there any known issue with the Operation helper?

Reply:
All this should be `Operations`, with `s`! But that isn't the problem, is it?

Reply:
Oops, forgot the `s` multiple times, I added it everywhere, thanks.
No, this is not the problem...

Reply:
I would be surprised if that's (still) valid. Did you try with a recent version?

Reply:
I tried with v7r0p36